src/components/ProductCard/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Link from 'next/link'
+
+const ProductCard = () => {
+  return (
+    <div className="product-card">
+      <Link href="/product-view">
```

Comment:
Since we are going to pass some props to each product item. I would consider use `Next router` instead

Reply:
This PR is actually for ProductView but the thing here is just a mockup so it could be implemented in future.